ibm/config.go

```diff
@@ -931,6 +939,19 @@ func (c *Config) ClientSession() (interface{}, error) {
 		session.directlinkErr = fmt.Errorf("Error occured while configuring Direct Link Service: %s", session.directlinkErr)
 	}
 
+	//Direct link provider
+	directLinkProviderV2Options := &dlProviderV2.DirectLinkProviderV2Options{
+		URL: envFallBack([]string{"IBMCLOUD_DIRECT_LINK_PROVIDER_URL"}, "https://directlink.cloud.ibm.com/provider/v2"),
```

Comment:
change IBMCLOUD_DIRECT_LINK_PROVIDER_URL to IBMCLOUD_DL_PROVIDER_API_ENDPOINT
Similar to dl one